index.js

```diff
@@ -4,7 +4,42 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+// Place holders
+let num = 0
+command = process.argv[2];
+// error handling
+//  as long as .length is less than or equal to 2
+// Logic : ()
+if (process.argv.length <= 2) {
+// return No OP error.
+    return "No operation provided..."
+// plus || minus are provided but no Number 
+// Logic : ()
+}       else if (process.argv.length <= 3){
+// return No Numbers error.
+            return "No numbers provided..."
+};
+
+
+// plus
+if (command === "plus"){
+    for (i =3; i < process.argv.length; i++){
```

Comment:
always use `let` or `const` when you declare a new variable - even inside a for loop!